zerver/lib/export.py

```diff
@@ -1296,12 +1296,29 @@ def convert_to_id_fields(data: TableData, table: TableName, field_name: Field) -
         item[field_name + "_id"] = item[field_name]
         del item[field_name]
 
-def re_map_foreign_keys(data_table: List[Record],
+def re_map_foreign_keys(data: TableData,
+                        table: TableName,
                         field_name: Field,
                         related_table: TableName,
                         verbose: bool=False,
                         id_field: bool=False,
                         recipient_field: bool=False) -> None:
+    """
+    This is a wrapper function for all the realm data tables
+    and only avatar and attachment records need to be passed through the internal function
+    because of the difference in data format (TableData corresponding to realm data tables
+    and List[Record] corresponding to the avatar and attachment records)
+    """
+    re_map_foreign_keys_internal(data[table], table, field_name, related_table, verbose, id_field,
```

Comment:
I have used the wrapper function logic here mentioned in the review of the PR #8683. Sorry for missing this one!